src/debt.py

```
@@ -50,3 +50,22 @@ def get_debts_for_service_resource(service_id: str):
     :return:
     """
     return api_caller.call_get(f'/services/{service_id}/debt')
+
+
+@debt_mcp.tool(annotations={"title": "Create Debt"})
+def create_debt(service_id: str, title: str, description: str, type: str):
```

Ruff (0.15.12)

[error] Function argument `type` is shadowing a Python builtin

(A002)